src/components/HelloWorld.vue

```diff
@@ -43,13 +43,13 @@
         <input type="button" value="1" @click="input('1')">
         <input type="button" value="2" @click="input('2')">
         <input type="button" value="3" @click="input('3')">
-        <input type="button" value="*" @click="input('*')">
+        <input type="button" value="*" @click="inputOprator('*')">
```

Comment:
```suggestion
        <input type="button" value="*" @click="inputOperator('*')">
```
오타 있네여... 제가 한거긴 한데 수정해주세여...